src/main/java/cubicchunks/world/ICubeCache.java

```diff
@@ -23,14 +23,19 @@
  */
 package cubicchunks.world;
 
+import cubicchunks.util.CubeCoords;
 import cubicchunks.world.column.Column;
 import cubicchunks.world.cube.Cube;
 
 public interface ICubeCache {
 	boolean cubeExists(int cubeX, int cubeY, int cubeZ);
 
+	boolean cubeExists(CubeCoords coords);
```

Comment:
I don't understand quite what you mean. In both ServerCubeCache and ClientCubeCache this method acts as a proxy for cubeExists(x, y, z). Therefor on the client it always returns true as required.